test/Application/ConfigInjectionTest.php

```diff
@@ -99,6 +134,44 @@ public static function assertPipelineContainsInstanceOf($class, $pipeline, $mess
         Assert::assertThat($found, Assert::isTrue(), $message);
     }
 
+    public function assertRouteMiddleware(MiddlewareInterface $middleware)
```

Comment:
Other custom assertion in that file are `static`.

Reply:
Missed that; fixing now.